packages/xchain-bitcoincash/src/client.ts

```diff
@@ -289,8 +292,9 @@ class Client implements BitcoinCashClient, XChainClient {
    *
    * @throws {"Invalid address"} Thrown if the given address is an invalid address.
    */
-  getBalance = async (address?: string): Promise<Balance[]> => {
-    return utils.getBalance({ haskoinUrl: this.getHaskoinURL(), address: address || this.getAddress() })
+  getBalance = async (index = 0): Promise<Balance[]> => {
```

Comment:
Hmm I don't think we should do that for `getBalance`

we should keep `getBalance(address?: string` imo

cc. @veado